kernels/available/python/default.nix

```diff
@@ -12,29 +12,52 @@
   projectDir ? self + "/kernels/available/python",
   pyproject ? projectDir + "/pyproject.toml",
   poetrylock ? projectDir + "/poetry.lock",
-  overrides ? poetry2nix.overrides.withDefaults (import ./overrides.nix),
+  overrides ? import ./overrides.nix,
   python ? pkgs.python3,
   editablePackageSources ? {},
+  runtimePackages ?
+    with pkgs; [
+      bashInteractive
+      coreutils
+    ],
+  extraRuntimePackages ? with pkgs; [sox],
```

Comment:
Defaults for `runtimePackages` and `extraRuntimePackages` should be empty lists, `[ ]`. Sometimes `runtimePackages` are non-empty by default if there is something necessary for the kernel to operate.

Generally, we put required packages in `runtimePackages` and user-desired packages in `extraRuntimePackages` which is why it is always left empty by default. But both are exposed so the user can modify them.

Reply:
These are indeed wrong (should be empty as you say). I pushed them because I was still confused while making tests.